tests/test_definitions/column_map_expectations/expect_column_values_to_be_between.json

```diff
@@ -450,7 +450,7 @@
             "column": "x",
             "max_value": 20,
             "min_value": 0,
-            "mostly": 0.9
```

Comment:
This test case was identical to the one above it functionally. Now it adds value by confirming that mostly can be set to higher precision.